de_web_plugin.cpp

```diff
@@ -7354,12 +7355,7 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node, const SensorFi
     else if (modelId.startsWith(QLatin1String("lumi")))
     {
         sensorNode.setManufacturer("LUMI");
-        if (!sensorNode.modelId().startsWith(QLatin1String("lumi.ctrl_")) &&
-            !sensorNode.modelId().startsWith(QLatin1String("lumi.plug")) &&
-            !sensorNode.modelId().startsWith(QLatin1String("lumi.relay.c")) &&
-            sensorNode.modelId() != QLatin1String("lumi.curtain") &&
-            sensorNode.modelId() != QLatin1String("lumi.sensor_natgas") &&
-            sensorNode.modelId() != QLatin1String("lumi.switch.b1naus01") &&
+        if (!sensorNode.node()->nodeDescriptor().receiverOnWhenIdle() && sensorNode.modelId() != QLatin1String("lumi.sensor_natgas") &&
```

Comment:
Is it safe to remove all those `modelId` checks?

I'm not 100% certain that the Node Descriptor is always valid at this point.

Given:

    node->nodeDescriptor().isNull()

This is always true:

    !sensorNode.node()->nodeDescriptor().receiverOnWhenIdle()

Reply:
As discussed, happy to move away from the ND for safety reasons and list the model IDs individually for now.

Reply:
Reverted